wiki/Ranking_Criteria/id.md

```diff
@@ -1,17 +1,12 @@
----
-outdated: true
-outdated_since: f2a2541a7f3463857bb378af0c07814ad7faf405
----
+# Ranking criteria
 
-# Kriteria ranking
+*Untuk ranking criteria spesifik [mode permainan](/wiki/Game_mode), kunjungi: [osu!](osu!), [osu!taiko](osu!taiko), [osu!catch](osu!catch), dan [osu!mania](osu!mania)*
```

Comment:
```suggestion
*Untuk kriteria ranking [mode permainan](/wiki/Game_mode) yang spesifik, kunjungi: [osu!](osu!), [osu!taiko](osu!taiko), [osu!catch](osu!catch), dan [osu!mania](osu!mania)*
```

Reply:
this refers to the line above
if you found similar case (to be `kriteria ranking`), please fix